tests/client_test.py

```diff
@@ -199,6 +199,31 @@ def test_get_client_kwargs_custom_endpoint(self):
                     'login_customer_id': None,
                     'logging_config': None
                 })
+    def test_load_from_dict(self):
+        config = {
+            'developer_token': self.developer_token,
+            'client_id': self.client_id,
+            'client_secret': self.client_secret,
+            'refresh_token': self.refresh_token}
+        mock_credentials_instance = mock.Mock()
+
+        with mock.patch.object(
+            Client.GoogleAdsClient,
+            '__init__',
+            return_value=None
+        ) as mock_client_init, mock.patch.object(
+            Client.oauth2,
+            'get_installed_app_credentials',
+            return_value=mock_credentials_instance
+        ) as mock_credentials:
+          Client.GoogleAdsClient.load_from_dict(config)
+          mock_client_init.assert_called_once_with(
+          credentials=mock_credentials_instance,
+          developer_token=self.developer_token,
+          endpoint=None,
+          login_customer_id=None,
+          logging_config=None)
```

Comment:
Lines 219 and 220 should be indented by 4 spaces, lines 221 - 225 should be indented by another 4, so:

```python
def test_load_from_dict(self):
    ...
    mock_client_init.assert_called_once_with(
        credentials=mock_credentials_instance,
        ...)
```